cleanlab/rank.py

```diff
@@ -109,57 +123,335 @@ def get_self_confidence_for_each_label(
       The columns must be ordered such that these probabilities correspond to class 0,1,2,...
       `pred_probs` should have been computed using 3 (or higher) fold cross-validation.
 
+    method : {"self_confidence", "normalized_margin", "confidence_weighted_entropy"}, default="self_confidence"
+      Label quality scoring method. Default is "self_confidence".
+
+      .. seealso::
+        :func:`self_confidence`
+        :func:`normalized_margin`
+        :func:`confidence_weighted_entropy`
+
+    adj_pred_probs : bool, default = True
```

Comment:
The default is `False` currently. If False is kept as default, then docstring should explain "Set this = True if you prefer..." rather than "Set this = False if ..."

Reply:
Good catch. I changed the default to True.

Reply:
Once we decide on default scoring method for the functions, we'll also want to make sure `adj_pred_probs` default matches across:  
 `order_label_issues()`, `score_label_quality()`, `score_label_quality_ensemble()`

Reply:
Confirming that I changed all defaults to:
`method="normalized_margin"`
`adj_pred_probs=False`

eb9c57a04b33412f6a547860e816d21b9d6c05ca